packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -613,23 +597,17 @@ export const startSentry = (runtimeConfiguration: RuntimeConfiguration, app: App
       attachProps: true,
       logErrors: true,
       ...moreSentryOptions
-    })
+    } as SentryConfig)
```

Comment:
... why is this needed?

Reply:
Without I get: `The types of 'transportOptions.fetchOptions.window' are incompatible between these types. Type 'unknown' is not assignable to type 'null'.`, which weird, because the `SentryConfig` is of the same type as the function's argument. But I don't know how or why this happens.

Reply:
odd, we can look together